crates/jstz_mock/src/lib.rs

```diff
@@ -1 +1,49 @@
-pub mod mock;
+use jstz_crypto::hash::Blake2b;
+use tezos_crypto_rs::hash::ContractKt1Hash;
+use tezos_smart_rollup::{
+    michelson::{
+        ticket::{FA2_1Ticket, Ticket},
+        MichelsonBytes, MichelsonNat, MichelsonOption, MichelsonPair,
+    },
+    types::Contract,
+};
+
+pub mod host;
+pub mod message;
+
+pub fn parse_ticket(
+    ticketer: ContractKt1Hash,
+    amount: u32,
+    content: (u32, Option<Vec<u8>>),
+) -> FA2_1Ticket {
+    let ticket_content = MichelsonPair(
+        MichelsonNat::from(content.0),
+        MichelsonOption::<MichelsonBytes>(content.1.clone().map(MichelsonBytes)),
+    );
+
+    Ticket::new(
+        Contract::Originated(ticketer.clone()),
```

Comment:
nit: I think most of the `.clone`s here can be avoided. Destructuring content should avoid the clone above, and I think you can simply move `ticketer` since it's owned and not used elsewhere

Reply:
`.clone`s removed